pkg/bundle/sync.go

```diff
@@ -130,7 +131,12 @@ func (b *bundle) buildSourceBundle(ctx context.Context, bundle *trustapi.Bundle)
 		return bundleData{}, fmt.Errorf("couldn't find any valid certificates in bundle")
 	}
 
-	if err := resolvedBundle.populateData(bundles, bundle.Spec.Target); err != nil {
+	deduplicatedBundles, err := deduplicateBundles(bundles)
```

Comment:
Just a small comment, can we also add an sorting/ordering mechanism to ensure it always generates deterministic bundle even if the input order is different? This ensures the system stability.

Reply:
Which key should be used for sort?